### PR TITLE
New Grafana panel using other Kubewarden metrics.

### PR DESCRIPTION
**Pull request by jvanz**
Adds three more panels using the latest metrics added in Kubewarden. The metrics are the policy counter and policy evaluation latency metrics.

### CODE REVIEW

**Comment by jvanz** [comment]
Screenshot from 2021-11-17 17-28-40

File: kubewarden-dashboard.json [hunk 1]
```diff
@@ -65,8 +23,8 @@
   "fiscalYearStartMonth": 0,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": null,
```
Comment (kkaempf) [inline kubewarden-dashboard.json:68]: What is `id` and why is 26 the right value ?

**Review by kkaempf — APPROVED** [review]
lgtm

File: kubewarden-dashboard.json (hunk 1 above)
Comment (jvanz) [reply kubewarden-dashboard.json:68]: I do not know. I believe Grafana did this change when I've exported the dashboard.

**Pull request merged**